lib/index.js

```diff
@@ -51,3 +50,23 @@ function fetchDinner(date) {
 function fetchSnacks(date) {
   $.getJSON(createDiaryURL(date, 'snacks'), createFoodHandler('#snacks'));
 }
+
+function fetchDiaryInfo() {
+  $('tbody.foods').html('');
+  fetchBreakfast(currentDate);
+  fetchLunch(currentDate);
+  fetchDinner(currentDate);
+  fetchSnacks(currentDate);
+}
+
+function previousDay() {
+  currentDate = currentDate.subtract(1, "days");
```

Comment:
This is a method available from momentjs to do calculations on dates. It makes my heart go 💥